website/templates/public/login.mako

```diff
@@ -60,34 +73,49 @@
             method="POST"
             data-bind="submit: submit"
         >
-            <h3 class="m-b-lg"> Login </h3>
+        <h3 class=${'m-b-lg' if not existing_user else 'm-b-lg m-l-md'}> Login </h3>
+        <div class="form-group">
+            <label for="inputEmail3" class="col-sm-2 control-label">Email</label>
```

Comment:
I tweaked the form pretty significantly to increase aesthetics and usability.  

Before
-
<img width="1162" alt="screen shot 2016-04-19 at 3 47 58 pm" src="https://cloud.githubusercontent.com/assets/6232068/14653266/3be7fc92-0646-11e6-9672-1a478027b1a7.png">

After
-

<img width="1294" alt="screen shot 2016-04-19 at 3 48 34 pm" src="https://cloud.githubusercontent.com/assets/6232068/14653273/414a965e-0646-11e6-8c3e-10e9b41bccd2.png">